elastic/integrations
# Update index template mapping

## What does this PR do?



## Checklist


## Author's Checklist



## How to test this PR locally
Install the integration and verify ES is able to index the cloudbeat events according to the new mapping. 


## Related issues


-

## Screenshots
